### PR TITLE
Update doc performance table and limitations

### PR DESCRIPTION
**Pull request by satyaog**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -16,38 +16,47 @@
...
 .. Note::
-   * Intel Xeon E5-2623 is the Xeon E5-2623 v3 @ 3.00 GHz version
-   * Tesla V100 is the Tesla V100 PCIE 16GB version
+   * Intel Xeon 2698 is the Intel Xeon E5-2698 v4 @ 2.20GHz version
+   * Tesla V100 is the Tesla V100 SXM2 16GB version
 
-The name "Benzina" is a phonetic transliteration of the Ukrainian word "Бензина", meaning "gasoline" (or "petrol").
+While DALI currently outperforms Benzina, the speedup can only be seen on JPEGs
+through the `nvJPEG <https://developer.nvidia.com/nvjpeg>`_ decoder. Benzina
+requires to transcode the input dataset to H.265 but then the gain can be seen
```
Comment (nouiz) [inline README.rst:44]: Great to see this advancing.
Did you check that this conversion doesn't impact training?

Comment (satyaog) [reply README.rst:44]: Hey sorry for the delay, what you describe is our current step. We will perform a statistical analysis to evaluate the accuracy between the image formats

Comment (nouiz) [reply README.rst:44]: Sound good. I'll continue to watch this repo:)

**Comment by satyaog** [comment]
Closed in favour of #31

**Pull request closed without merging**